src/requests/cache/basic/get.rs

```diff
@@ -58,8 +58,9 @@ impl<K: IntoBytes> MomentoRequest for GetRequest<K> {
     type Response = Get;
 
     async fn send(self, cache_client: &CacheClient) -> MomentoResult<Get> {
-        let request = prep_request(
+        let request = prep_request_with_timeout(
```

Comment:
oh this is interesting.

I'm not sure we should even *have* a `prep_request` function, it kinda seems like we should *only* have the `_with_timeout` one. WDYT? (not an issue for this PR but something we may want to come back and clean up.)

Reply:
definitely!